charm-slurmd/actions.yaml

```diff
@@ -1,3 +1,5 @@
+version:
+  description: Return version of installed software
```

Comment:
What are your thoughts on replicating this action in the other charms?

It returns something like:
```
$ juju run-action slurmd/leader version --wait
unit-slurmd-94:
  UnitId: slurmd/94
  id: "1116"
  results:
    infiniband: not installed
    munge: 0.5.14-4
    nhc: 1.4.2-omni-1.0
    slurm: 20.11.4-1
  status: completed
  timing:
    completed: 2021-05-27 17:04:34 +0000 UTC
    enqueued: 2021-05-27 17:04:30 +0000 UTC
    started: 2021-05-27 17:04:34 +0000 UTC
```